### DISCUSSION TITLE
Feature request: notifications for comments and mentions under a gist

### DISCUSSION THREAD

**Discussion by ilyaigpetrov** [community/community#21721] [New to GitHub]
<p>GitHub Gists may be used not only for sharing snippets of code but as a complete publishing platform for those who prefer markdown and unobtrusive sharing of their gist articles.</p>
<p>However for many years neither authors nor commenters receive notifications about discussion under a gist.</p>
<p>Please, <strong>add notifications for comments and replies under gists</strong>.</p>

**Comment by BelHFL** [comment]
<p>Hi <a class="mention" href="/ilyaigpetrov">@ilyaigpetrov</a>,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Cheers!</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>You can now receive notifications for new conversations occurring on gists. Similar to issues and pull requests, you can opt out of these notifications at any time. Learn more about <a href="https://help.github.com/en/articles/about-gists#receiving-notifications-for-gist-activity" rel="nofollow noopener">gist notifications.</a></p>
